pom.xml

```diff
@@ -342,6 +348,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <id>run-ant-examples</id>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <phase>verify</phase>
+            <configuration>
+              <!-- ensure the examples stay compatible with evolving main changes -->
+              <target name="examples">
+                <ant antfile="${basedir}/examples/example1/build.xml" dir="${basedir}/examples/example1" target="compileAndClean" />
+                <ant antfile="${basedir}/examples/example2/build.xml" dir="${basedir}/examples/example2" target="compileAndClean" />
+                <!--ant antfile="${basedir}/examples/example3/build.xml" dir="${basedir}/examples/example3" target="installAndClean"/-->
```

Comment:
So example 3 and 5 are not needed? then why create it?

Reply:
Good catch! My bad, they are valid examples. I'll update.